ironfish-rust-nodejs/index.d.ts

```diff
@@ -33,6 +33,10 @@ export interface NativeSpendDescription {
   nullifier: Buffer
 }
 export const PROOF_LENGTH: number
+export const TRANSACTION_SIGNATURE_LENGTH: number
```

Comment:
Can you split all of these changes out into another PR? They are not related to removing `workerPool.createTransaction()` right and they are fairly opinionated so mat or rohan should review them separately.

Reply:
That sounds good, but it's still unrelated to this PR. It's not needed to accomplish the PR task. It should be pretty easy to split this out into a different PR.

Reply:
Yeah @ygao76 I think this is a sensible change; I'm all for removing magic numbers. But as @NullSoldier mentioned, we want to keep the code changes in this PR focused on just the worker pool. I'd recommend keeping this refactoring but just cherry picking those changes out in a new PR.